Gruntfile.js

```diff
@@ -60,10 +60,10 @@ module.exports = function(grunt) {
         ],
         tasks: [
           'jshint',
-          'clean:dev',
-          'compass:dev',
+          'clean:dist',
+          'compass:dist',
           'handlebars',
-          'concat:dev'
+          'concat:dist'
```

Comment:
It does seem like overkill for watch.  We need to add it to the build task too. I'm assuming it wasn't there before because the  images were committed and they didn't think they would be changing.